pkg/processor/build/builder.go

```diff
@@ -516,6 +516,11 @@ func (b *Builder) validateAndEnrichConfiguration() error {
 		b.options.FunctionConfig.Spec.Runtime = "python:3.9"
 	}
 
+	// enrich project name
+	if _, err := b.options.FunctionConfig.GetProjectName(); err != nil {
```

Comment:
I noticed we also have `b.GetProjectName()` in the builder, we might want to align the functionality of it with the `Config`'s same function.
What I mean is that we don't really need both, or at least the builder can use the config's method.

Reply:
@TomerShor good catch, I removed `GetProjectName()` in builder, so we use the one from function config